### DISCUSSION TITLE
Do we expect paramiko 3.0 to work with Vorta?

### DISCUSSION THREAD

**Discussion by Hofer-Julian** [borgbase/vorta#1558] [FAQ]
The bot opened a PR to update the paramiko library in the Vorta flatpak to 3.0.
Since that is a major release, I wanted to ask if we expect this to work before I merge it.

https://github.com/flathub/com.borgbase.Vorta/pull/94

**Comment by m3nu** [comment]
We only use it to verify SSH keys. Just updated and didn't notice any issues.

Maybe there is a quick- and dirty way to parse SSH keys with the standard library and drop this dependency?

Finally, since this update only came out 11 hours ago, I wouldn't update it right away.

**Reply by Hofer-Julian** [reply]
> Finally, since this update only came out 11 hours ago, I wouldn't update it right away.

Okay, what sounds like a reasonable time frame to you?

**Reply by m3nu** [reply]
2 weeks maybe?

But just removing it sounds even better. There were errors parsing SSH keys in the past and we don't need to be super-strict about it. If we only show files that start with `-----BEGIN OPENSSH PRIVATE KEY-----` in the `~/.ssh` folder that would work too and drop Paramiko altogether.

**Reply by Hofer-Julian** [reply]
> 2 weeks maybe?

👍🏻 

> But just removing it sounds even better.

Sounds good!
I will probably not be the one to open the PR though :)
